airbyte-cdk/bulk/core/load/src/integrationTest/kotlin/io/airbyte/cdk/load/mock_integration_test/MockBasicFunctionalityIntegrationTest.kt

```diff
@@ -91,4 +92,6 @@ class MockBasicFunctionalityIntegrationTest :
     override fun testBasicTypes() {
         super.testBasicTypes()
     }
+
+    @Test @Disabled override fun testBasicWriteFile() {}
```

Comment:
Nit: Maybe leave a comment about why we are disabling.

Reply:
I think that it is not needed for the file use case. The mocked stream loader for the file doesn't bring more value than the record one.